### PR TITLE
Bug fixes and add TR to final score

### PR DESCRIPTION
**Pull request by vincentneko**
- Add TR to final score
- Fix final greenery bug
- Fix WaterSplittingPlant missing canAct
- Fix Tardigrades missing canAct

### CODE REVIEW

File: src/Game.ts [hunk 1]
```diff
@@ -390,6 +390,11 @@ export class Game {
     private gotoEndGame(): void {
         this.phase = Phase.END;
 
+        // TR is converted in victory points
+        this.players.forEach((player) => {
+            player.victoryPoints += player.terraformRating;
```
Comment (bafolts) [inline src/Game.ts]: Does the outside of the board represent a player's terraform rating or victory points? Does this make more sense to do the opposite and add victoryPoints to terraformRating?

File: src/cards/WaterSplittingPlant.ts [hunk 2]
```diff
@@ -19,6 +19,9 @@ export class WaterSplittingPlant implements IProjectCard {
         return undefined;
     }
     public actionText: string = "Spend 3 energy to increase oxygen level by 1";
+    public canAct(player: Player): boolean {
+        return player.energy >= 3;
+    }
     public action(player: Player, game: Game) {
         if (player.energy < 3) {
```
Comment (bafolts) [inline src/cards/WaterSplittingPlant.ts]: I would remove this `throw "Need 3 energy"`. I was throwing these errors before I added the `canAct` method.

Comment (vincentneko) [reply src/cards/WaterSplittingPlant.ts]: done

File: src/Game.ts (hunk 1 above)
Comment (vincentneko) [reply src/Game.ts]: The answer is both. Before end game the outside of the board represents a player's terraform rating but after end game, it represents victory points.
Scoring is based on victory points so I guess it make more sense to add TR to VP in the end. (TR is only one way to score VPs)

**Pull request merged**